packages/image_picker/image_picker/test/image_picker_deprecated_test.dart

```diff
@@ -57,14 +57,16 @@ void main() {
                 'maxWidth': null,
                 'maxHeight': null,
                 'imageQuality': null,
-                'cameraDevice': 0
+                'cameraDevice': 0,
+                'requestFullMetadata': true,
```

Comment:
You'll need to reconcile the test changes against https://github.com/flutter/plugins/pull/5706 once that lands (which is needed to resolve the current tree breakage, so will be before this PR can land).